teos/cli/teos_cli.py

```diff
@@ -110,85 +155,267 @@ def stop(self):
-    except Exception as e:
-        sys.exit(f"Unknown error occurred: {str(e)}")
+@Cli._register_command
+class GetAllAppointmentsCommand(CliCommand):
+    """
+    NAME:   teos-cli get_all_appointments - Gets information about all the appointments stored in the tower.
+
+    USAGE:  teos-cli get_all_appointments
+
+    DESCRIPTION:
+
+        Gets information about all appointments stored in the tower.
+    """
+
+    name = "get_all_appointments"
+
+    @staticmethod
+    def run(rpc_client, opts_args):
+        return rpc_client.get_all_appointments()
+
+
+@Cli._register_command
+class GetTowerInfoCommand(CliCommand):
+    """
+    NAME:   teos-cli get_tower_info - Gets generic information about the tower.
+
+    USAGE:  teos-cli get_tower_info
+
+    DESCRIPTION:
+
+        Gets generic information about the tower, like tower_id and aggregate data on users and appointments.
+    """
 
+    name = "get_tower_info"
+
+    @staticmethod
+    def run(rpc_client, opts_args):
+        return rpc_client.get_tower_info()
+
+
+@Cli._register_command
+class GetUsersCommand(CliCommand):
+    """
+    NAME:   teos-cli get_users - Gets the list of registered user ids.
+
+    USAGE:  teos-cli get_users
+
+    DESCRIPTION:
+
+        Gets an array with the user ids of all the users registered to the tower.
+    """
+
+    name = "get_users"
+
+    @staticmethod
+    def run(rpc_client, opts_args):
+        return rpc_client.get_users()
+
+
+@Cli._register_command
+class GetUserCommand(CliCommand):
+    """
+    NAME:   teos-cli get_user - Gets information about a specific user.
+
+    USAGE:  teos-cli get_user "user_id"
+
+    DESCRIPTION:
+
+        Gets information about a specific user.
+    """
+
+    name = "get_user"
+
+    @staticmethod
+    def run(rpc_client, opts_args):
+        opts, args = opts_args
+
+        if not args:
+            sys.exit("No user_id was given")
+        if len(args) > 1:
+            sys.exit(f"Expected only one argument, not {len(args)}")
+
+        return rpc_client.get_user(args[0])
+
+
+@Cli._register_command
+class StopCommand(CliCommand):
+    """
+    NAME:   teos-cli stop - Requests a graceful shutdown of the tower..
+
+    USAGE:  teos-cli stop
+
+    DESCRIPTION:
+
+        Requests a graceful shutdown of the tower.
+    """
+
+    name = "stop"
+
+    @staticmethod
+    def run(rpc_client, opts_args):
+        rpc_client.stop()
+        print("Closing the Eye of Satoshi")
+
+
+@Cli._register_command
+class HelpCommand(CliCommand):
+    """
+    NAME:   teos-cli help - Shows general help, or help for a specific command.
+
+    USAGE:  teos-cli help [command]
+
+    DESCRIPTION:
+
+        Shows a summary of all the commands and global options. If command is given, shows detailed help for the
+        command.
+    """
+
+    name = "help"
+
+    @staticmethod
+    def run(rpc_client, opts_args):
+        opts, args = opts_args
+
+        print(opts_args)
```

Comment:
Remove this print?

Reply:
Oops, removed in 26dcfcf591ed3c63b0aae95a524007493b8044ce.